docs/first-order/receptive_fields.ipynb

```diff
@@ -23,7 +23,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        unit_rf = np.zeros([ys.size, xs.size])
</pre><p>good useful description. try to put comments on the same tab level as the line they're describing otherwise it might be unclear. maybe say "for each x and y coordinate" to be very clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>